utils/scap-as-rpm.8

```diff
@@ -13,8 +13,9 @@ FILE [FILE ...]
 .PP
 Takes given SCAP input(s) and makes an RPM package that contains them. The
 result RPM can be installed using # yum install ./package\-name\-1\-1.rpm which
-will put the contents into \fI/usr/share/xml/scap\fP. No dependency on openscap or scapworkbench is enforced in the output package so you can use any SCAP\-capable
-scanner to evaluate the content.
+will put the contents into \fI/usr/share/xml/scap\fP. No dependency on openscap
```

Comment:
```suggestion
will put the contents into \fI/usr/share/xml/scap\fP. No dependency on the OpenSCAP
```

Reply:
Updated with correct capitalization on OpenSCAP, but couldn't get 'the' to fit in.